ui-v2/tests/acceptance/dc/acls/tokens/index.feature

```diff
@@ -77,7 +77,7 @@ Feature: dc / acls / tokens / index: ACL Token List
     s: Si-Search
     ---
     And I see 1 token model
-    And I see 1 token model with the serviceIdentity "Si-Search"
+    And I see 1 token model with the serviceIdentity "Service Identity: Si-Search"
```

Comment:
Again, I'm not massively happy with this change as it depends on the design of the service identity display. I decided to PR this instead of getting deeper into this and adding more and more change here.